mlflow/sagemaker/cli.py

```diff
@@ -9,6 +9,7 @@
 from mlflow.sagemaker import DEFAULT_IMAGE_NAME as IMAGE
 from mlflow.utils import cli_args
 from mlflow.utils import env_manager as em
+from mlflow.utils.file_utils import TempDir
```

Comment:
can we use the built-in `tempfile`?

Reply:
I think `tempfile.TemporaryDirectory` is more universal than `mlflow.utils.file_utils. TempDir`. I know what `TempDir` does but that might not be the case for new contributors or new hires.

Reply:
Not a blocker

Reply:
Makes a lot sense, thanks for the explanation!